src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -193,8 +193,7 @@
         <property name="lookupableHelperService">
             <ref bean="userProcurementProfileLookupableHelperService" />
         </property>
-    </bean>
-
+    </bean>        
```

Comment:
there are no relevant changes to this file so it would be good to revert to original version